docs/azure-tre-features/airlock.md

```diff
@@ -0,0 +1,130 @@
+# Airlock
+
+In a Trusted Research Environment (TRE) the workspaces represent a security boundary that enables researchers to access data, execute analysis, apply algorithms and collect reports. The airlock capability brings the only mechanisme that allows users to `import` or `export` data and tools in a secure fashion with a manual approval.
+This constitutes the mechanism focused in preventing malware inside TRE and/or TRE workspaces and preventing data exfiltration, while allowing researchers to execute their tasks.
+The airlock feature brings several actions: ingress/egress Mechanism; Data movement; Security gates; Approval mechanism and Notifications. As part of TRE's Safe settings all activity must be tracked for auditiing purpose.
+
+The Airlock feature aims to address these goals:
+* Prevent unauthorised data export (or import).
+* Provide a process to allow approved data into the security boundary of a TRE Workspace.
+* TRE provides functionality to track requests and decisions, supporting cycle of revision, approval or rejection.
+* Data being imported with an airlock import process must be automatically scanned for security purpose.
+* Data bein exported or imported must ne manually reviewed bu the Workspace Owner.
+* Notify the involved users of the process progress and/or required actions
+* All airlock processes are audited and tracked
+
+Typically in a TRE, the Airlock feature would be used to allow a researcher to export patient identifiers for an exact cohort without the underlying data ever being exposed. Through the airlock, data to be exported is restricted by a manual review ensuring that no data is exfiltered from TRE.
+
+The Airlock feature will create events on every miningful step of the processes. This will enable increased flexibility by allowing an organization to extend several underlying mechanisms.
+
+## Ingress/Egress Mechanism
+
+The Ingress/Egress Mechanism allows a TRE user to start `import` or `export` process to a given workspace. The mechanism provides a set of milestones that must be reached in order to execute a process end to end. This milestones end up to be states:
+
+a. **Draft**: An import/export process has been created but has not yet started. The TRE User/Researcher has now access to a storage location and he must identify the data to be processed. At this point the airlock import/export processes allow a single file to be processed. However a compressed file may be used (zip).
+b. **Submitted**: The request was submitted by the ressearcher (not yet processed).
+c. **In-Review**: The request was submitted is in review.
```

Comment:
```suggestion
c. **In-Review**: The request is ready to be reviewed. This state can be reached directly from Submitted state or after being scanned for malware (and found clean) 
```

Reply:
THanks. Just rephrased it to remove malware scanning (for now). Using the term security scan